Updated docs to reflect change to ncWMS 2.2.8

guygriffiths opened the pull request.

kalxas commented:
Thanks for the review.
Cameron has also suggested to add "Things to try" and "What next" sections at end, like other projects do.
Please check ncwms (row 34 at https://docs.google.com/spreadsheets/d/1Q5BaEgQtgw4O1bXyeWMlM8XtAOhUgcjZ7Y2O0FZc2H0/edit#gid=2014800150)

cvvergara commented:
@guygriffiths 

I am checking the documentation generation.
There is this problem

```
/tmp/osgeolive_make/en/overview/ncWMS_overview.rst:: WARNING: image file not readable: images/screenshots/1024x768/ncWMS-03-timeseries.png
```
https://travis-ci.org/cvvergara/OSGeoLive-doc/builds/244959785#L634


The image  `ncWMS-03-timeseries.png`  is missing .
```
$ ls images/screenshots/1024x768/nc*
images/screenshots/1024x768/ncWMS-01-start_screen.png
images/screenshots/1024x768/ncWMS-02-variable_view.png
images/screenshots/1024x768/ncWMS-04-north_pole.png
```
https://github.com/OSGeo/OSGeoLive-doc/tree/master/images/screenshots/1024x768

Can you please make a PR with the missing `ncWMS-03-timeseries.png`

guygriffiths commented:
@cvvergara Apologies, I had removed the image because it was no longer used in the quickstart guide.  I've changed the overview to use a different image now. I'll open another PR with the change
